akka-projection-testkit/src/test/scala/akka/projection/testkit/scaladsl/ProjectionTestKitSpec.scala

```diff
@@ -122,14 +123,29 @@ class ProjectionTestKitSpec extends ScalaTestWithActorTestKit with AnyWordSpecLi
       val strBuffer = new StringBuffer("")
       val projection = TestProjection(Source(1 to 5), strBuffer, _ <= 5)
 
-      val sinkProbe = projectionTestKit.runWithTestSink(projection)
-
-      sinkProbe.request(5)
-      sinkProbe.expectNextN(5)
-      sinkProbe.expectComplete()
+      projectionTestKit.runWithTestSink(projection) { sinkProbe =>
+        sinkProbe.request(5)
+        sinkProbe.expectNextN(5)
+        sinkProbe.expectComplete()
+      }
 
       strBuffer.toString shouldBe "1-2-3-4-5"
     }
+
+    "run cray" in {
```

Comment:
`run cray` ?

https://www.urbandictionary.com/define.php?term=cray%20cray

Test doesn't look that bad ;)

Reply:
You won't believe me, but I don't know from that comes. I probably add it to a run a dumb test and forgot to remove. I don't remember to have ever typed `run cray` also not `run crazy`.

Reply:
hmm, I think I remember now. It was emitting 1 element (like crazy) and cancelling the probe. I was verifying that I could cancel it and trigger the stopHandler. Anyway, that's cray!